### PR TITLE
Document how to launch a build

### PR DESCRIPTION
**Pull request by camzeech**
This is part one of the user guide short recast.
Part two can be done in a separate ticket.
I am not sure if the rephrasing of the title of the page (and the :training) breaks hyperlinks, so I'm happy to be corrected.
Ideally @yias if you can contribute to this branch and directly add a couple of steps on the "how to build", as I halted where it was, after project creation & data upload. Can you add one or two steps around "how to build" + add to the references at the bottom of the page links to the news methods / objects you created to build from the SDK?

### CODE REVIEW

File: doc/source/user_guide/training.rst [hunk 1]
```diff
@@ -45,9 +44,6 @@ Train on prediction data
 
      td.add_to_project(project)
 
-Once you have training data in your project, you can use the web app to
-train a model.
-
```
Comment (romane-m) [inline doc/source/user_guide/training.rst:50]: Maybe this:
Your training data is imported and assigned to a project. (_optional:_ You can now configure your AI model.)
If the _optional_ next step used, an hyperlink to the model config api reference is to be added at the end of this page.

File: doc/source/user_guide/training.rst [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
-Before you can use the solver, you must train the SimAI solution on your prediction
-data. You first upload your prediction data into a global pool of
-:class:`training data<ansys.simai.core.data.training_data.TrainingData>` instances
+SimAI allows you to build AI models on your simulation data. This first step is to upload your simulation data
+into a global pool of :class:`training data<ansys.simai.core.data.training_data.TrainingData>` instances
 and then assign this data to different :class:`Project<ansys.simai.core.data.projects.Project>`
-instances, which you configure for training your model.
+instances, which you configure in order to build your model.
 
-Train on prediction data
```
Comment (romane-m) [inline doc/source/user_guide/training.rst:19]: The second step of this procedure might need to be rephrased as well:

#. Upload your prediction data by creating a
   :class:`TrainingData<ansys.simai.core.data.training_data.TrainingData>` instance
   and then loading your files into it::
   
   #. Create a :class:`TrainingData<ansys.simai.core.data.training_data.TrainingData>` instance
   and upload your simulation data/results into it::

**Review by romane-m — COMMENTED** [review]
Feel free to integrate or not :)

Obviously, this feedback is just to better the current doc. A more extensive effort is to be planned on the pysimai User Guide :)

**Comment by yias** [comment]
Example on how to launch a build is added. @romane-m could you please review?
fyi @camzeech

**Review by yias — APPROVED** [review]

**Pull request merged**